vault/src/system.rs

```diff
@@ -478,7 +479,30 @@ impl VaultService {
         }
     }
 
+    async fn ensure_bitcoin_networks_match(&self) -> Result<(), Error> {
+        let bitcoin_network = self.btc_rpc_master_wallet.network().to_string();
+        let default_spec_bitcoin_network = &Value::default();
+        let system_properties = self.btc_parachain.get_rpc_properties().await.unwrap_or_default();
+        let parachain_bitcoin_network = system_properties
+            .get("bitcoinNetwork")
+            .unwrap_or(default_spec_bitcoin_network)
```

Comment:
```suggestion
            .unwrap_or_default()
```

Reply:
I tried that and got this error, probably because system_properties.get("bitcoinNetwork") is a reference option:
 ```
 error[E0277]: the trait bound `&jsonrpc_core::Value: Default` is not satisfied
   --> vault/src/system.rs:488:14
    |
488 |             .unwrap_or_default()
    |              ^^^^^^^^^^^^^^^^^ the trait `Default` is not implemented for `&jsonrpc_core::Value`
    |
    = help: the following implementations were found:
              <jsonrpc_core::Value as Default>
note: required by a bound in `std::option::Option::<T>::unwrap_or_default`
   --> /home/dan/.rustup/toolchains/nightly-2022-02-08-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:837:12
    |
837 |         T: ~const Default,
    |            ^^^^^^^^^^^^^^ required by this bound in `std::option::Option::<T>::unwrap_or_default`

For more information about this error, try `rustc --explain E0277`.
```